spotify/main.py

```diff
@@ -54,11 +54,29 @@ def load_file_or_env(var, default=None):
 
 # Start- and end-date for the data we want to fetch
 # Load from environment variable if set, otherwise default to current date
-START_DATE = os.environ.get("START_DATE", (dt.datetime.now() - dt.timedelta(days=4)).strftime("%Y-%m-%d"))
-END_DATE = os.environ.get("END_DATE", (dt.datetime.now() - dt.timedelta(days=1)).strftime("%Y-%m-%d"))
+START_DATE = os.environ.get(
+    "START_DATE", (dt.datetime.now() - dt.timedelta(days=4)).strftime("%Y-%m-%d")
+)
+END_DATE = os.environ.get(
+    "END_DATE", (dt.datetime.now() - dt.timedelta(days=1)).strftime("%Y-%m-%d")
+)
 
-print("Done initializing environment")
+# List of endpoints to fetch data from
+# We split them into two groups because some endpoint names are identical
+# This has the added benefit that we can completely skip fetching episode data
+# by setting `EPISODE_ENDPOINTS` to an empty string
+DEFAULT_PODCAST_ENDPOINTS="metadata,detailedStreams,listeners,aggregate,followers,episodes"
```

Comment:
what happens if episodes is not defined? there won't be any episodes for the loop over episodes?